src/components/FindWallet/WalletTable/WalletMoreInfoCategory.tsx

```diff
@@ -29,35 +36,40 @@ export const WalletMoreInfoCategory = ({
   const { t } = useTranslation("page-wallets-find-wallet")
 
   return (
-    <Box width="full" mt={4} _first={{ mt: 2 }}>
-      <Heading
-        as="h4"
-        lineHeight={1.4}
-        fontSize="md"
-        fontWeight={500}
-        mx="0.2rem"
-        mb={2}
-      >
+    <Box mx={{ base: 0, md: 0, lg: 2 }} w="100%">
+      {/* Category label */}
+      <Heading as="h4" lineHeight={1.4} fontSize="md" fontWeight="bold" mb={2}>
```

Comment:
```suggestion
      <Heading as="h4" size="md" mb={2}>
```

Reply:
`size="md"` is not the same font size defined [in the design](https://www.figma.com/file/rv850lIICwOwRnyqSdtZED/Redesign-find-wallet?type=design&node-id=2289-7095&mode=design&t=Wlxi1fSerY50efXv-0) @konopkja